internal/api/create.go

```diff
@@ -27,12 +27,12 @@ type CreateResponse struct {
 }
 
 type CreateHandler struct {
-	db  db.DB
-	bus *bus.EventBus
+	DB  db.DB
```

Comment:
why make these public?

Reply:
because of the end_to_end test. No biggy if it shouldn't be